components/app/myrw/datasets/pages/my-rw-datasets/dataset-list/dataset-list-component.js

```diff
@@ -18,7 +18,10 @@ class DatasetsList extends PureComponent {
     },
     datasets: [],
     filters: [],
-    loading: true
+    loading: true,
+    user: {},
+    currentTab: '',
+    getDatasetsByTab: () => null
```

Comment:
Let's try to avoid this kind of things. Usually, this kind of functions are mandatory so better adding an `isRequired` attribute and avoids confusion.

In case of a function is really not required, better asking in the code if it exists instead of executing it.